restart_process/Tiltfile

```diff
@@ -22,15 +22,14 @@ _CUSTOM_BUILD_KWARGS_BLACKLIST = [
 _ext_dir = os.getcwd()
 
 # shared code between the two restart functions
-def _helper(base_ref, ref, entrypoint, live_update, restart_file=RESTART_FILE, trigger=None, **kwargs):
+def _helper(base_ref, ref, entrypoint, live_update, restart_file=RESTART_FILE, trigger=None, entr_flags=None, **kwargs):
     if not trigger:
         trigger = []
 
     # declare a new docker build that adds a static binary of tilt-restart-wrapper
     # (which makes use of `entr` to watch files and restart processes) to the user's image
     df = '''
-    FROM tiltdev/restart-helper:2020-10-16 as restart-helper
-
+    FROM tiltdev/restart-helper:2021-08-05v2 as restart-helper
```

Comment:
We will need to push a new version of the restart helper, then once it is up I can update this to the right value.

Reply:
This is also why the tests are failing
```  failing-job │ ERROR: Build Failed: ImageBuild: failed to create LLB definition: docker.io/tiltdev/restart-helper:2021-08-05v2: not found```